codefresh/resource_pipeline_test.go

```diff
@@ -731,6 +731,57 @@ resource "codefresh_pipeline" "test" {
 `, rName, originalYamlString)
 }
 
+func testAccCodefreshPipelineBasicConfigDescription(rName, repo, path, revision, context, description string) string {
+	return fmt.Sprintf(`
+resource "codefresh_pipeline" "test" {
+
+  lifecycle {
+    ignore_changes = [
+      revision
+    ]
+  }
+
+  name = "%s"
+
+  spec {
+	spec_template {
+		repo        = %q
+		path        = %q
+		revision    = %q
+		context     = %q
+	}
+
+	description = "%s"
+  }
+}
+`, rName, repo, path, revision, context, description)
+}
+
+func TestAccCodefreshPipeline_Description(t *testing.T) {
```

Comment:
Are you sure this test is valid? You just added the description property under the `spec_template` and here you are specifying it under the `spec`